benchmark/fs/bench-existsSync.js

```diff
@@ -0,0 +1,25 @@
+'use strict';
+
+const common = require('../common');
+const fs = require('fs');
+const tmpdir = require('../../test/common/tmpdir');
+tmpdir.refresh();
+
+const paths = [
+  __filename,
+  tmpdir.resolve(`.non-existing-file-${process.pid}`),
+];
+
+const bench = common.createBenchmark(main, {
+  n: [1e6],
```

Comment:
You can put the paths here as a parameter so that in the benchmark results the performance of operations on existing/non-existent paths can be categorized. Also the parameters should include an existing path that are returned by a path method (i.e. not flat) to reflect real world usage, I think the improvement is probably largely from `__filename` which is flat. But normally paths aren't flat if they are computed by some path method.

Reply:
I've updated the benchmarks:

```
                                                           confidence improvement accuracy (*)   (**)  (***)
fs/bench-existsSync.js n=1000000 type='existing'                 ***      2.45 %       ±1.36% ±1.81% ±2.36%
fs/bench-existsSync.js n=1000000 type='non-existing'             ***     30.63 %       ±1.49% ±1.99% ±2.59%
fs/bench-existsSync.js n=1000000 type='non-flat-existing'         **      2.57 %       ±1.52% ±2.02% ±2.64%

Be aware that when doing many comparisons the risk of a false-positive result increases.
In this case, there are 3 comparisons, you can thus expect the following amount of false-positive results:
  0.15 false positives, when considering a   5% risk acceptance (*, **, ***),
  0.03 false positives, when considering a   1% risk acceptance (**, ***),
  0.00 false positives, when considering a 0.1% risk acceptance (***)
```